src/reducers/signupSignupReducer.js

```diff
@@ -1,12 +1,37 @@
+import {
+  USERNAME_CHANGED,
+  EMAIL_CHANGED,
+  PASSWORD_CHANGED,
+  CONF_PASSWORD_CHANGED,
+  ERROR,
+} from '../actions/types';
+
 const initialState = {
   loading: false,
   registerError: '',
   loginError: '',
 };
 
-export default (state = initialState, action) => {
+const signupSignin = (state = initialState, action) => {
```

Comment:
Why dispatch an action for each form field change? Are you not using the component state instead of this?

Reply:
this file was from a bad rebase but @ezrogha rectified this in his PR